### PR TITLE
added controller tests for comments

### PR DESCRIPTION
**Pull request by maria-kulakova**

### CODE REVIEW

File: test/controllers/comments_controller_test.rb [hunk 1]
```diff
@@ -1,29 +1,50 @@
...
+    @comment.save
+    get edit_recipe_comment_path(@recipe.id, @comment.id)
     assert_response :success
   end
 
-  test "should get update" do
-    get comments_update_url
-    assert_response :success
+  test "should update comment" do
+    post "/recipes/#{recipes(:one).id}/comments", params: { comment: { text: "!!!", recipe_id: recipes(:one).id } }
```
Comment (kiosan) [inline test/controllers/comments_controller_test.rb]: use route helpers

File: test/controllers/comments_controller_test.rb [hunk 2]
```diff
@@ -1,29 +1,50 @@
...
+    assert_redirected_to recipe_path(recipes(:one))
   end
 
-  test "should get destroy" do
-    get comments_destroy_url
-    assert_response :success
+  test "should destroy comment" do
+    assert_difference('Comment.count', -1) do
+      delete "/recipes/#{recipes(:one).id}/comments/#{comments(:one).id}"
```
Comment (kiosan) [inline test/controllers/comments_controller_test.rb]: delete receipt_comment_path

File: test/system/recipes_test.rb [hunk 3]
```diff
@@ -8,7 +8,7 @@ def setup
     @recipe = recipes(:one)
     @recipe.user = users(:one)
     @recipe.save
-    binding.pry
+    # binding.pry
```
Comment (kiosan) [inline test/system/recipes_test.rb]: remove, not comment

File: test/controllers/comments_controller_test.rb (hunk 1 above)
Comment (maria-kulakova) [reply test/controllers/comments_controller_test.rb]: Thanks, I will make that changes

File: test/system/recipes_test.rb (hunk 3 above)
Comment (maria-kulakova) [reply test/system/recipes_test.rb]: I remove it

File: test/controllers/comments_controller_test.rb (hunk 2 above)
Comment (maria-kulakova) [reply test/controllers/comments_controller_test.rb]: Thanks for correcting, I'll change it

**Pull request merged**